src/rust/ide/view/src/project.rs

```diff
@@ -57,8 +58,14 @@ impl Model {
     }
 
     fn set_style(&self, is_light:bool) {
-        if is_light { self.app.themes.set_enabled(&["dark"])  }
-        else        { self.app.themes.set_enabled(&["light"]) }
+        let root_elem = web::get_element_by_id("root").unwrap_or_else(|_| panic!("Failed to find root!"));
+        if is_light {
+            self.app.themes.set_enabled(&["dark"]);
+            root_elem.set_class_name("dark");
+        } else {
+            self.app.themes.set_enabled(&["light"]);
+            root_elem.set_class_name("");
+        }
```

Comment:
1. I don't like setting `.dark` ondark theme and nothing on light theme. Lets make it uniform. Lets set `.light` on light theme as well.

2. I'm not sure I like `.dark` and `.like` names. Maybe letsm ake them a little bit more explicit - what about `.dark-theme` and `.light-theme`? 

3. This does not work in `debug/interface` - everything should. Please check it always there (!)

4. Copy-pasting this code to `debug/interface` is error prone, thus create functions for it: `self.app.themes.set_dark_enabled();`etc

Reply:
4 is not necessary and it works both in IDE and in interface.